libshaderc/include/shaderc.h

```diff
@@ -141,6 +141,46 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_forced_version_profile(
     shaderc_compile_options_t options, int version, shaderc_profile profile);
 
+// To support file inclusion, libshaderc invokes a callback into its client to
+// resolve the full path and content of the included file.
+// Client callback should follow the specified function signature below, and
+// it should be passed to libshaderc through the corresponding setter function.
+// When the including of a file is done, libshaderc will call another client
+// callback to clean up the resources used for the including process.  Client
+// should implement the clean up method and pass it to libshaderc together with
+// the response method.
+
+// The struct that contains the information to be returned to the includer.
+// The client-side implemented response method should return this struct. And
+// the underlying data is owned by client code.
+struct shaderc_includer_response {
+  const char* path;
+  size_t path_length;
+  const char* content;
+  size_t content_length;
+};
+
+// The function signature of the client-side implemented reponse method. It
+// returns shaderc_includer_reponsestruct.
```

Comment:
..."returns a pointer to a shaderc_includer_response."

Missed the pointer part.